emoji-ui/src/components/sourcelist/sourcelist.js

```diff
@@ -27,8 +28,8 @@ class SourceListBase extends Component {
 
   render() {
     const { sourceEmoji } = this.props;
-    const firstTwenty = sourceEmoji.slice(0, 20);
-    return <h1>{JSON.stringify(firstTwenty)}</h1>;
+    const firstBatch = sourceEmoji.slice(0, 200);
```

Comment:
Are we planning on adding pagination on the front or backend?

Reply:
yea, i am thinking about working on pagination next if no one is going to work on it